hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -2222,6 +2225,43 @@ public List<OzoneFileStatus> listStatus(OmKeyArgs args, boolean recursive,
     return statusList;
   }
 
+  @Override
+  public List<OzoneFileStatusLight> listStatusLight(OmKeyArgs args,
+      boolean recursive, String startKey, long numEntries,
+      boolean allowPartialPrefixes) throws IOException {
+    KeyArgs keyArgs = KeyArgs.newBuilder()
+        .setVolumeName(args.getVolumeName())
+        .setBucketName(args.getBucketName())
+        .setKeyName(args.getKeyName())
+        .setSortDatanodes(args.getSortDatanodes())
+        .setLatestVersionLocation(args.getLatestVersionLocation())
```

Comment:
If `BasicKeyInfo` does not include key locations, are these necessary?  I wonder if we should hard-code `setSortDatanodes(false)` and `setLatestVersionLocation(true)`?

(also applies to `OzoneManagerRequestHandler`)

Reply:
Yes, `listStatusLight` [`BasicKeyInfo`] does not include `KeyLocations` and I believe hard-coding these values should help, thank you!